datalad/distribution/install.py

```diff
@@ -285,8 +282,8 @@ class Install(Interface):
     @staticmethod
     @datasetmethod(name='install')
     def __call__(
+            source,
             path=None,
-            source=None,
```

Comment:
This is how you got me the first time, but this time I am prepared!

The apt comparison is inadequate: It is even in the name: apt-GET